package.json

```diff
@@ -115,7 +115,7 @@
     "files": [
       {
         "path": "dist/main.js",
-        "maxSize": "400KB"
+        "maxSize": "450KB"
```

Comment:
Bumped this up because the [Lua highlighting mode](https://github.com/codemirror/CodeMirror/blob/master/mode/lua/lua.js), along with the recent addition of GraphQL highlighting, seems to have pushed it over the edge.

We can drastically reduce the bundle size by enabling minimization but I'm not sure what that'll break. Before and after with `minimize: true` in the main browser export:

![Screen Shot 2021-12-09 at 3 34 30 PM](https://user-images.githubusercontent.com/33762/145492398-3e19204a-cfa9-4f05-a4ce-d6e55c341b6f.png)